commands/login.go

```diff
@@ -67,4 +62,7 @@ func (cmd *Login) Run(scope scope.Scope, c *cli.Context) {
 
 	cmd.config.Map["target"] = cmd.network.BrooklynUrl
 	cmd.config.Write()
+	
```

Comment:
Maybe do the login before writing the config and only do the latter if the former succeeds.